snuba/utils/streams/backends/kafka.py

```diff
@@ -629,6 +629,16 @@ def closed(self) -> bool:
     "sasl.username",
     "sasl.password",
     "security.protocol",
+    "ssl.client.auth",
+    "ssl.endpoint.identification.algorithm",
+    "ssl.key.location",
+    "ssl.key.password",
+    "ssl.key.pem",
+    "ssl.keystore.location",
+    "ssl.keystore.password",
+    "ssl.truststore.location",
+    "ssl.truststore.password",
```

Comment:
Same I cannot find these ones in thge librdkafka settings. Do these parameters actually work ?

Reply:
I'm actually not sure what the relationship is between the list of global configuration properties and all of the kafka settings that I see in client configuration guides. For example, [configuring clients with SSL](https://docs.confluent.io/platform/current/kafka/authentication_ssl.html#clients) recommends setting `ssl.truststore.location`, `ssl.truststore.password`, and (optionally) `ssl.truststore.type`:

```
bootstrap.servers=kafka1:9093
security.protocol=SSL
ssl.truststore.location=/var/private/ssl/kafka.client.truststore.jks
ssl.truststore.password=test1234
ssl.keystore.location=/var/private/ssl/kafka.client.keystore.jks
ssl.keystore.password=test1234
ssl.key.password=test1234
```

Reply:
(I've also asked the owners of the `librdkafka` settings list for more context. Hopefully I'll have a better answer for you soon.)

Reply:
My only concern was to add to this list only setting keys that actually work. Do you have a chance to test that on your broker? If it works and it takes those settings into account, I am ok with that.
Otherwise if you do not receive an answer very soon, let me know, I think we can move ahead.

Reply:
The extra settings I added only apply to the Kafka Java client, so I removed them. Now just adding the SSL configurations in the `librdkafka` settings list.